index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="flex-container outer-container">
+        <header id="navbar" class="flex-container">
+            <!--- Item to display logo -->
+            <div class="flex-item" id="logo">
+                Welcome
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">Portfolio</a>
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">About</a>
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">Contact</a>
+            </div>
+            <div class="flex-item" id="link">
+                <i class="fa-solid fa-bars"></i></i>
+            </div>
+        </header>
```

Comment:
- [OPTIONAL] Kindly consider having two direct children in your `header` tag. one is `a` tag for logo. and second is `nav` tag in which you add the hamburger menu. this is due to following best practices and convention and following best semantics and user experience.